app/controllers/navigation/Navigator.scala

```diff
@@ -122,7 +122,7 @@ object Navigator {
 
   val occasional: PartialFunction[DeclarationPage, Mode => Call] = {
     case BorderTransport             => controllers.declaration.routes.SupervisingCustomsOfficeController.displayPage
```

Comment:
Can you remove in that case BorderTransport? user shouldn't be there

Reply:
Good spot!.   Removing it for simp and occ simp